### PR TITLE
added Chinese translation of contributor cheat sheet

### PR DESCRIPTION
**Pull request by idealhack**
fixes https://github.com/kubernetes/website/issues/13990

this PR also added links for translations and cleaned up some links and typos

### CODE REVIEW

**Comment by mrbobbytables** [comment]
Thanks @idealhack  \o/

**Comment by chenrui333** [comment]
Nice job. Will check out later.

File: contributors/guide/contributor-cheatsheet/README-id.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ di GitHub menjadi lebih baik.
 - [kubernetes-dev] - Alamat email pengembangan Kubernetes
 - [Forum Kubernetes] - Forum resmi Kubernetes.
 - [Slack _channels_] - Slack resmi Kubernetes.
-- [StackOverflow] - _Platform_ tanya jawab pengguna Kubernetes.
+- [Stack Overflow] - _Platform_ tanya jawab pengguna Kubernetes.
```
Comment (chenrui333) [inline contributors/guide/contributor-cheatsheet/README-id.md]: 👍

File: contributors/guide/contributor-cheatsheet/README-id.md [hunk 2]
```diff
@@ -185,7 +185,7 @@ dokumentasi, atau segala bentuk hal yang disimpan dalam repositori git.
 - [Label]
 - [Perintah Prow][perintah bot]
 - [Pull request process]
-- [Github workflow]
+- [GitHub workflow]
```
Comment (chenrui333) [inline contributors/guide/contributor-cheatsheet/README-id.md]: 👍

File: contributors/guide/contributor-cheatsheet/README-zh.md [hunk 3]
```diff
@@ -0,0 +1,328 @@
+# Kubernetes 贡献者备忘单
+
+这是一份参与 Kubernetes 协作时的常用资源清单，包含 Kubernetes 项目中常用的提示、技巧和最佳实践。这个「长话短说」或快速参考包含一些有用的信息，能使您的 GitHub 贡献体验更好。
```
Comment (chenrui333) [inline contributors/guide/contributor-cheatsheet/README-zh.md]:
```suggestion
这是一份参与 Kubernetes 协作时的常用资源清单，包含 Kubernetes 项目中常用的提示、技巧和最佳实践。这个「长话短说」或快速参考包含一些有用的信息，能使您有更好的 GitHub 贡献体验。
```

File: contributors/guide/contributor-cheatsheet/README-zh.md [hunk 4]
```diff
@@ -0,0 +1,328 @@
...
+
+- [主要小组列表][sigs]
+
+### 社区
+
+- [日历] - 查看所有的 Kubernetes 社区事件（SIG 或 WG 会议、活动等）
+- [kubernetes-dev] - Kubernetes 开发邮件列表
+- [Kubernetes 论坛] - Kubernetes 官方论坛
+- [Slack 频道] - Kubernetes 官方 Slack
+- [Stack Overflow] - Kubernetes 最终用户提问的地方
```
Comment (chenrui333) [inline contributors/guide/contributor-cheatsheet/README-zh.md]: 用户或者使用者
```suggestion
- [Stack Overflow] - Kubernetes 用户提问的地方
```

File: contributors/guide/contributor-cheatsheet/README-zh.md [hunk 5]
```diff
@@ -0,0 +1,328 @@
...
+#### 回应 Issue
+
+- 在处理 issue 时，添加评论以便让其他人知道，避免重复的工作。
+- 如果之后您解决了什么问题，在关闭前该 issue 前，先评论以便让其他人知道。
+- 添加其它 PR 或 issue 的引用（或者任何可能的资料），例如 _「ref: #1234」_。这对于联系其它地方处理过的相关工作很有帮助。
+
+
+### 提交 Pull Request
+
+Pull requests（PR）是提交代码、文档或其它形式的工作的主要途径，这些工作都保存在 git 存储库中。
```
Comment (chenrui333) [inline contributors/guide/contributor-cheatsheet/README-zh.md]: repo 应该是 仓库，或者代码仓库更为准确一点。
```suggestion
Pull requests（PR）是提交代码、文档或其它形式的工作的主要途径，这些工作都保存在 git 代码仓库中。
```

File: contributors/guide/contributor-cheatsheet/README-zh.md [hunk 6]
```diff
@@ -0,0 +1,328 @@
...
+/sig contributor-experience
+/cc @stakeholder1 @stakeholder2
+/kind cleanup
+/area developer-guide
+/assign @approver1 @approver2 @approver3
+```
+
+这个 PR 里包含了：
+
+- **第 1 行** - 提及其它有关的 issue 或 PR（#3064 #3097）
```
Comment (chenrui333) [inline contributors/guide/contributor-cheatsheet/README-zh.md]: nitpick comment。 末尾的句号。:smile:

File: contributors/guide/contributor-cheatsheet/README-zh.md [hunk 7]
```diff
@@ -0,0 +1,328 @@
...
+- **第 2 行** - 对该 PR 做了什么事情的一个简短的描述
```
Comment (chenrui333) [inline contributors/guide/contributor-cheatsheet/README-zh.md]:
```suggestion
- **第 2 行** - 对该 PR 做了什么事情的一个简短的描述。
```

File: contributors/guide/contributor-cheatsheet/README-zh.md [hunk 8]
```diff
@@ -0,0 +1,328 @@
...
+- **第 4 行** - 使用 `/sig contributor-experience` [命令]分配 [SIG][sigs]
```
Comment (chenrui333) [inline contributors/guide/contributor-cheatsheet/README-zh.md]:
```suggestion
- **第 4 行** - 使用 `/sig contributor-experience` [命令]分配 [SIG][sigs]。
```

File: contributors/guide/contributor-cheatsheet/README-zh.md (hunk 6 above)
Comment (chenrui333) [reply contributors/guide/contributor-cheatsheet/README-zh.md]: 后面多行也都是。

File: contributors/guide/contributor-cheatsheet/README-zh.md [hunk 9]
```diff
@@ -0,0 +1,328 @@
...
+[故障排除指南]: https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/
+[Kubernetes 论坛]: https://discuss.kubernetes.io/
+[Pull request 处理]: /contributors/guide/pull-requests.md
+[GitHub 工作流程]: /contributors/guide/github-workflow.md
+[prow]: https://git.k8s.io/test-infra/prow#prow
+[cla]: /CLA.md#how-do-i-sign
+[cla 故障排除指南]: /CLA.md#troubleshooting
+[命令]: https://prow.k8s.io/command-help
+[kind]: https://prow.k8s.io/command-help#kind
+[cc]: https://prow.k8s.io/command-help#hold
```
Comment (chenrui333) [inline contributors/guide/contributor-cheatsheet/README-zh.md]: 原文的错误。
https://prow.k8s.io/command-help#cc

File: contributors/guide/contributor-cheatsheet/README-zh.md [hunk 10]
```diff
@@ -0,0 +1,328 @@
...
+- [Atlassian git 教程]
+- [Git 魔法]
+- [GitHub 工作流程]
+- [本地测试]
+- [开发者指南]
+
+
+### 分支策略
+
+Kubernetes 项目使用 GitHub 标准的 _「Fork 然后 Pull」_ 的工作流程。在 git 术语中，您个人的 fork 被称为 _「`origin`」_，实际的项目 git 存储库被称为 _「`upstream`」_。为了保证您的个人分支（`origin`）与项目（`upstream`）保持更新，必须要在您的本地工作副本中进行设置。
```
Comment (chenrui333) [inline contributors/guide/contributor-cheatsheet/README-zh.md]:
```suggestion
Kubernetes 项目使用 GitHub 标准的 _「Fork 然后 Pull」_ 的工作流程。在 git 术语中，您个人的 fork 被称为 _「`origin`」_，实际的项目 git 仓库被称为 _「`upstream`」_。为了保证您的个人分支（`origin`）与项目（`upstream`）保持更新，必须要在您的本地工作副本中进行设置。
```

File: contributors/guide/contributor-cheatsheet/README-zh.md [hunk 11]
```diff
@@ -0,0 +1,328 @@
...
+
+
+#### 添加上游
+
+添加 `upstream` 作为远程分支，并将它设置中的 push 选项关闭。
```
Comment (chenrui333) [inline contributors/guide/contributor-cheatsheet/README-zh.md]: origin and upstream are all `remote`s.
 
```suggestion
添加 `upstream` 作为一个远程分支，并将它设置中的 push 选项关闭。
```

File: contributors/guide/contributor-cheatsheet/README-zh.md [hunk 12]
```diff
@@ -0,0 +1,328 @@
...
+git remote add upstream <upstream git repo>
+git remote set-url --push upstream no_push
+```
+
+您可以运行 `git remote -v` 列出设置过的远程分支来验证这一点。
+
+
+#### 让您的工作保持同步
+
+获取 `upstream` 的所有变更，然后将您本地的 `master` 分支在它基础上 _「rebase」_。这样您本地的存储库与 `upstream` 项目就能保持同步了。
```
Comment (chenrui333) [inline contributors/guide/contributor-cheatsheet/README-zh.md]:
```suggestion
获取 `upstream` 的所有变更，然后将您本地的 `master` 分支在它基础上 _「rebase」_。这样您本地仓库与 `upstream` 项目就能保持同步了。
```

**Comment by idealhack** [comment]
@chenrui333 thanks for your review, I think I've addressed all of them :)

**Pull request merged**